03-Custom-Actions-In-Python/01-What-are-Actions.ipynb

```diff
@@ -0,0 +1,121 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What are Actions?\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "### Questions\n",
+    "\n",
+    "- What are actions?\n",
+    "- Why would I want to use custom actions?\n",
+    "- What categories of actions exist for customization in HOOMD-blue?\n",
+    "\n",
+    "### Objectives\n",
+    "\n",
+    "- Explain the concept of an action in HOOMD-blue.\n",
+    "- Discuss potential use cases for custom Python actions.\n",
+    "- Provide the categories of actions in HOOMD-blue.\n",
+    "\n",
+    "## Actions\n",
+    "\n",
+    "Actions are the objects that _act_ or operate on a\n",
+    "`hoomd.Simulation` object. Looking at HOOMD-blue\n",
+    "from the Python perspective, actions are wrapped\n",
+    "or composed by operations. For example, objects like\n",
+    "`hoomd.update.BoxResize` and `hoomd.hpmc.tune.MoveSize` contain\n",
+    "actions internally. In other words, the actions implement the logic \n",
+    "repsonsible for implementing the functionality of an operation\n",
+    "(e.g. resizing the box for `hoomd.update.BoxResize`), while the\n",
+    "operation handles some logistics like determining when the action\n",
+    "will run using a `hoomd.Trigger` object.\n",
+    "\n",
```

Comment:
This rewrite would help resolve my confusion about action vs. operation. I feel like it's more accessible to start with what a user knows already exist in hoomd.

```suggestion
    "Some HOOMD-blue operations you may have already used, like\n",
    "`hoomd.update.BoxResize` or `hoomd.hpmc.tune.MoveSize`, contain\n",
    "actions internally.\n",
    "The actions implement logic \n",
    "responsible for implementing the functionality of the operation\n",
    "(e.g. resizing the box for `hoomd.update.BoxResize`), while the\n",
    "operation handles some logistics like determining when the action\n",
    "will run using a `hoomd.Trigger` object.\n",
    "From the Python perspective, we say that actions are wrapped\n",
    "or composed by operations.\n",
    "\n",
```